sdk/tests/deqp/framework/opengl/gluShaderUtil.js

```diff
@@ -37,6 +37,21 @@ gluShaderUtil.GLSLVersion = {
 };
 
 /**
+ * glsUniformBlockCase.isGLSLVersionSupported
+ * @param {WebGL2RenderingContext|WebGLRenderingContextBase} ctx
+ * @param {gluShaderUtil.GLSLVersion} version
+ * @return {boolean}
+ */
+ gluShaderUtil.isGLSLVersionSupported = function(ctx, version) {
+    if (ctx instanceof WebGL2RenderingContext)
+        return version <= gluShaderUtil.GLSLVersion.V300_ES;
+    else if (ctx instanceof WebGLRenderingContextBase)
```

Comment:
This should not work in any browser. WebGLRenderingContextBase has the [NoInterfaceObject] extended attribute which means that its type is not visible. This should be WebGLRenderingContext instead.